operational_analysis/methods/plant_analysis.py

```diff
@@ -825,21 +830,22 @@ def sample_long_term_reanalysis(self, n, r):
            r(:obj:`string`): The reanalysis product used for the current Monte Carlo iteration
         Returns:
            :obj:`pandas.DataFrame`: the windiness-corrected or 'long-term' annualized monthly/daily wind speeds
+           :obj:`pandas.DataFrame`: Long-term wind speed stdev, from which IAV will be derived
         """
+        # Check if valid data has already been calculated and stored. If so, just return it
+        if (r, n) in self.long_term_sampling:
+            long_term_reg_inputs = self.long_term_sampling[(r, n)].copy()
+            ws_aggregate_sd = self.long_term_sampling_sd[(r, n)].copy()
```

Comment:
Is there a good reason to perform a copy here? It will increase the memory usage of the program.